### PR TITLE
Add Testing Details In Contributors Guide

### LINKED ISSUE #5878: Add Testing Details in Contributors Guide
Testing details are missing from the [Getting Started Contributors Guide](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/contributors/getting-started.md).

### PR DESCRIPTION
**Pull request by tarunvijwani**
Fixes #5878

This PR adds the testing details in the [Getting Started Contributors Guide](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/contributors/getting-started.md).

### Testing

Read the [Getting Started Contributors Guide](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/contributors/getting-started.md) and make sure testing steps are added correctly.

### CODE REVIEW

File: docs/contributors/getting-started.md [hunk 1]
```diff
@@ -125,6 +125,20 @@ To use Prettier, you should install the [Prettier - Code formatter](https://mark
...
 
+## Testing
+
+You’ll find a handful of scripts in `package.json` that performs the automated tests and linting. You can run the following commands to execute automated tests in your terminal:
+
+- JS tests: `npm run test`
+- End-to-End `tests:npm run test:e2e` 
+- PHP tests: `npm run test:php` 
+
+You can also use [Docker](https://www.docker.com/) to run end-to-end or php tests.
```
Comment (opr) [inline docs/contributors/getting-started.md]: I don't think this should be here, the npm scripts for PHP and E2E run in Docker anyway, maybe we should just skip this line, keep the line below about `npm run wp-env` though 👍🏼

**Review by opr — CHANGES_REQUESTED** [review]
Thanks Tarun, just left a comment that will need to be addressed before we can merge this.

File: docs/contributors/getting-started.md (hunk 1 above)
Comment (tarunvijwani) [reply docs/contributors/getting-started.md]: Sure, I have removed those lines. 👍

**Review by opr — APPROVED** [review]
Thanks! Looks good now

**Pull request merged**